shotover-proxy/example-configs/cassandra-redis-cache/topology.yaml

```diff
@@ -9,10 +9,10 @@ chain_config:
         caching_schema:
           test_cache_keyspace_batch_insert.test_table:
             partition_key: [id]
-            range_key: [id]
+            range_key: []
           test_cache_keyspace_simple.test_table:
             partition_key: [id]
-            range_key: [id]
+            range_key: []
```

Comment:
What does this change do?

Reply:
I dont know what it used to do but I believe with the current changes partition_key and range_key just alters whether a column goes in the hash's key or the hash's field.
e.g.
`hset "table:'foo'" "* WHERE " $CACHE`
instead of 
`hset "table'" "* WHERE id=foo " $CACHE`
I dont actually know why we want this though.


However I'm pretty sure that having partition_key and range_key both containing `id` doesnt make sense, so better to just remove from range_key.